src/System.Net.Http/src/System/Net/Http/Managed/HttpSystemProxy.cs

```diff
@@ -0,0 +1,194 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Http;
+using System.Net;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+using SafeWinHttpHandle = Interop.WinHttp.SafeWinHttpHandle;
+
+namespace System.Net.Http
+{
+    internal sealed class HttpSystemProxy : IWebProxy, IDisposable
+    {
+        private readonly Uri _proxyUri;         // String URI for HTTPS requests
```

Comment:
What does "String URI" mean when the type of the field is a Uri rather than string?